scripts/hostcfgd

```diff
@@ -1465,6 +1466,77 @@ class MgmtIfaceCfg(object):
         self.mgmt_vrf_enabled = enabled
 
 
+class SyslogCfg:
+    SYSLOG_RATE_LIMIT_INTERVAL = 'rate_limit_interval'
+    SYSLOG_RATE_LIMIT_BURST = 'rate_limit_burst'
+    HOST_KEY = 'GLOBAL'
+
+    # syslog conf file path in docker
+    SYSLOG_CONF_PATH = '/etc/rsyslog.conf'
+
+    # Regular expressions to extract value from rsyslog.conf
+    INTERVAL_PATTERN = '.*SystemLogRateLimitInterval\s+(\d+).*'
+    BURST_PATTERN = '.*SystemLogRateLimitBurst\s+(\d+).*'
+
+    def __init__(self):
+        self.current_interval, self.current_burst = self.parse_syslog_conf()
+
+    def syslog_update(self, data):
+        """Update syslog related configuration
+
+        Args:
+            data (dict): CONFIG DB data: {<field_name>: <field_value>}
+        """
+        new_interval = '0'
```

Comment:
Any particular reason to have these as strings instead of integers?

Reply:
Because redis pass string to hostcfgd, hostcfgd need compare redis value with existing value